src/Control/Monad/Bayes/Inference/MCMC.hs

```diff
@@ -11,12 +11,17 @@
 -- Portability : GHC
 module Control.Monad.Bayes.Inference.MCMC where
 
-import Control.Monad.Bayes.Class (MonadSample)
+import Control.Monad.Bayes.Class
+import Control.Monad.Bayes.Sampler.Strict
 import qualified Control.Monad.Bayes.Traced.Basic as Basic
-import Control.Monad.Bayes.Traced.Common (burnIn)
+import Control.Monad.Bayes.Traced.Common
```

Comment:
I don't want to cause a religious war but I think for production code it's good to have explicit imports as then it's easier for a maintainer to see where a function comes from. Of course, during development this is a pain. Thoughts?

Reply:
I agree